ui/nuxeo-layout.js

```diff
@@ -149,6 +149,10 @@ import './nuxeo-error.js';
       return submittable;
     }
 
+    _isVisible(node) {
+      return node && node.offsetParent && node.offsetHeight > 0 && node.offsetWidth > 0;
```

Comment:
Looks good to me. I think we should leave a comment of why we are using this tricky: we want to exclude fallback content from slots (and also point to ELEMENTS-1393 for more details).